### ISSUE TITLE
Encoding error in bodhi.client.bindings.BodhiClient.update_str()

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2546]
Bastien Nocera [reported](https://bugzilla.redhat.com/show_bug.cgi?id=1585660) a traceback when creating an update that used the [‘](https://www.fileformat.info/info/unicode/char/2018/index.htm) character in the update text:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/bodhi/client/__init__.py", line 273, in new
    print_resp(resp, client)
  File "/usr/lib/python2.7/site-packages/bodhi/client/__init__.py", line 736, in print_resp
    click.echo(client.update_str(resp))
  File "/usr/lib/python2.7/site-packages/bodhi/client/bindings.py", line 712, in update_str
    buf = six.moves.cStringIO(update['notes'])
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 98: ordinal not in range(128)
```

**Comment by bowlofeggs** [comment]
This fix is planned for inclusion in the upcoming 3.11.0 release: https://github.com/fedora-infra/bodhi/pull/2709

**Comment by bowlofeggs** [comment]
Bodhi 3.11.0b5 has been deployed to staging:

https://bodhi.stg.fedoraproject.org

It is also available via Copr:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release

**Comment by bowlofeggs** [comment]
This fix is included in Bodhi 3.11.0.
